contracts/errors.rs

```diff
@@ -23,4 +23,14 @@ pub enum Error {
     ClaimedAmountIsBiggerThanAvailable,
     // The amount of multipliers per Beneficiary is not equal to the amount of periods
     InvalidMultipliersLength,
+    // The multiplier id does not exist
+    MultiplierNotFound,
+    // The multiplier is already deactivated
+    MultiplierAlreadyDeactivated,
+    // The multiplier is not deactivated
+    MultiplierNotDeactivated,
+    // The multiplier is not deactivated
+    MultiplierStillInUse,
```

Comment:
MultiplierNotDeactivated and MultiplierStillInUse mean the same thing. Join them